tests/integration/jit/function/test_span_events_are_created_upon_caught_exception.php

```diff
@@ -21,6 +21,8 @@
 newrelic.transaction_tracer.threshold = 0
 newrelic.span_events_enabled=1
 newrelic.cross_application_tracer.enabled = false
+display_errors=1
+log_errors=0
 error_reporting = E_ALL
```

Comment:
I don't think the error related ini settings are required. The 'baseline' for this test doesn't have them: https://github.com/newrelic/newrelic-php-agent/blob/acc7942cb672999fd87537be9aae508915bcc916/tests/integration/span_events/test_span_events_are_created_upon_caught_exception.php#L19-L24

Reply:
The tests here are not meant to necessarily be duplicates.

The INI settings are to be consistent with the rest of the `tracing` and `function` tests and to ensure the log settings are consistent as sometimes the setting on a particular machine have different log settings which could then lead to different test behavior.

Reply:
These INI settings are not relevant to this test.

Reply:
The INI values are not required for the test to run.  However, we can improve our tests.

Since these are tests, if an unexpected error occurs (especially in a complex situation like JIT entails), we want to be notified of the diagnostics whether or not the error notification setting is toggled on the system or whether or not the default has changed.

```
display_errors string
This determines whether errors should be printed to the screen as part of the output or if they should be hidden from the user.
```

error_reporting default has changed in the past and could in the future.
```
error_reporting int
Set the error reporting level. The parameter is either an integer representing a bit field, or named constants. The error_reporting levels and constants are described in [Predefined Constants](https://www.php.net/manual/en/errorfunc.constants.php), and in php.ini. To set at runtime, use the [error_reporting()](https://www.php.net/manual/en/function.error-reporting.php) function. See also the [display_errors](https://www.php.net/manual/en/errorfunc.configuration.php#ini.display-errors) directive.

The default value is E_ALL.

Prior to PHP 8.0.0, the default value was: E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED. This means diagnostics of level E_NOTICE, E_STRICT and E_DEPRECATED were not shown. 
```